agent/ecs_client/model/api/api-2.json

```diff
@@ -2114,6 +2115,7 @@
         "status":{"shape":"String"},
         "reason":{"shape":"String"},
         "containers":{"shape":"ContainerStateChanges"},
+        "managedAgents":{"shape":"ManagedAgentStateChanges"},
```

Comment:
is `ManagedAgentStateChanges` already in here?

Reply:
1564     "ManagedAgentStateChanges":{

it is.